### PR TITLE
node version fix

### PR DESCRIPTION
**Pull request by dhruv282**
changes in node version

### CODE REVIEW

**Comment by ibnesayeed** [comment]
First of all, I would much prefer it is fixed to work with the `latest` version of `node`.

Secondly, if for some reason that can't happen then I would prefer the value of the `NODE_TAG` `ARG` is updated and not inline `FROM` directive, because `ARG` is there to allow building the image with different versions of the base image at build time by supplying `--build-arg NODE_TAG=<some-node-version>` argument.

Finally, I would not specify the patch version (the third point of the version number) to receive any security fixes in the near future and use something like `11.11` instead as the default value.

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-ARG     NODE_TAG=latest
-FROM    node:$NODE_TAG
+#ARG     NODE_TAG=latest
```
Comment (ibnesayeed) [inline Dockerfile]:
```suggestion
ARG     NODE_TAG=11.11
```

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+FROM    node:11.11.0
```
Comment (ibnesayeed) [inline Dockerfile]:
```suggestion
FROM    node:$NODE_TAG
```

**Comment by ibnesayeed** [comment]
Please test these changes and let me know when it is ready to be merged. If the version `11.11` does not work then change it to `11.11.0` or any other version that seems to work for now.

**Comment by dhruv282** [comment]
I just tested the changes and 11.11 works fine, thank you!

**Comment by ibnesayeed** [comment]
The quick fix is deployed, you should check if everything is working as expected on the live site.

**Pull request merged**